src/Components/Styles/Categories.css

```diff
@@ -0,0 +1,59 @@
+.categories {
+    margin: 1rem;
+    background-color: var(--white);
+}
+
+.categoryList {
+    display: flex;
+    flex-wrap: wrap;
+}
+
+
+.categoryDiv {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    align-items: center;
+    gap: 2.5rem;
+    margin: 4rem;
+}
+
+.category {
+    background-color: var(--yellow-300);
+    border-radius: 1rem;
+    width: 10rem;
+    height: 8rem;
+    padding-left: 2rem;
+    padding-right: 2rem;
```

Comment:
You can use this in a single line padding command

Reply:
Ok, Thanks for pointing out. I'll correct it.